### ISSUE TITLE
Question about doc "How to enable Kafka Reporter"

### ISSUE THREAD

**Issue by Humbertzhang** [apache/skywalking#5515]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?

When I try to use kafka as reporter, i followed the instruction in [How-to-enable-kafka-reporter](https://github.com/apache/skywalking/blob/master/docs/en/setup/service-agent/java-agent/How-to-enable-kafka-reporter.md), but i think there's something is ambiguous.

The document tells me 
> Move the jar of the plugin from `optional-reporter-plugins` to `reporter-plugins` for activating.

When I enter the  directory  of my Skywalking Java Agent, i found there's no **reporter-plugins** directory in agent folder, so i thought i should create the directory, and then "Move the jar of the plugin from optional-reporter-plugins to reporter-plugins". But it still uses grpc to send messages, rather than using kafka. 
And then I try to move the kafka-reporter-plugin jar to plugins, and then i find it works. 

To sum up, I think the description of the document here is not very clear, may be it's better to change it to :
> Move the jar of the plugin from `optional-reporter-plugins` to `plugins` for activating.

**Comment by wu-sheng** [comment]
Pay the attention on version. Different features started on different versions.

**Comment by wu-sheng** [comment]
If you only mean about the folder name, doc fix pr is welcome.
